tasks/lammps/run.py

```diff
@@ -78,11 +74,20 @@ def _process_lammps_result(
 
 
 @task
-def faasm(ctx, repeats=1, nprocs=None, procrange=None):
+def faasm(ctx, bench, repeats=1, nprocs=None, procrange=None):
     """
     Run LAMMPS experiment on Faasm
     """
-    result_file = _init_csv_file("lammps_wasm.csv")
+    if bench not in BENCHMARKS:
+        print("Unrecognized benchmark: {}".format(bench))
+        print("Benchmark must be one in: {}".format(BENCHMARKS.keys()))
+        exit(1)
+
+    _bench = BENCHMARKS[bench]
+
+    result_file = _init_csv_file(
+        "lammps_wasm_{}.csv".format(_bench["out_file"])
```

Comment:
I think this will fail with benchmarks that don't have an `"out_file"` entry. Could do `_bench.get("out_file", bench)` to default to the benchmark name.